claat/fetch/remotehttp/remotehttp_test.go

```diff
@@ -0,0 +1,67 @@
+package remotehttp
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"testing"
+)
+
+func TestFetch(t *testing.T) {
+	type input struct {
```

Comment:
nit: Benefit of having these as named types vs anonymous `structs`? or their inner types since the structure is simple enough?

maybe something like this:
```go
tests := []struct {
    inStatusCode  int
    wantError     bool
}{
```

`out.ok` seems redundant since it means `true`. 

This format seems to fit more complicated tests with various inputs and outputs args.

Reply:
The in/out structure is a common pattern: http://go/gh/golang/go/wiki/TableDrivenTests

Reply:
right. I mainly saw it as an overkill for this case (these are the main things I think about... welp)

Reply:
I'd prefer "overkill" to using a nonstandard pattern.